lib/contentTypeParser.js

```diff
@@ -55,12 +58,15 @@ ContentTypeParser.prototype.add = function (contentType, opts, parserFn) {
     parserFn
   )
 
-  if (contentType === '*') {
-    this.parserList.push('')
+  if (contentTypeIsString && contentType === '*') {
     this.customParsers[''] = parser
   } else {
-    if (contentType !== 'application/json') {
-      this.parserList.unshift(contentType)
+    if (contentTypeIsString) {
+      if (contentType !== 'application/json' && contentType !== 'text/plain') {
```

Comment:
I added `text/plain` to this line because without this check the value `text/plain` is twice in the `parserList` array as soon as the user overrides the `plain/text` content type parser